### DISCUSSION TITLE
Questions from beginners

### DISCUSSION THREAD

**Discussion by zhiyoutech** [eclipse-tycho/tycho#2651] [Q&A]
1) what it the version relationship between tycho , maven and  JavaSE?
2) what are the options for some plugins, from example,  tycho-compiler-plugin.  there are some options for tycho-compiler-plugin, i do not know what is the meaning of <source>、<target> and <optimize>.
                    <plugin>
				<groupId>org.eclipse.tycho</groupId>
				<artifactId>tycho-compiler-plugin</artifactId>
				<configuration>
					<source>1.8</source>
					<target>1.8</target>
					<optimize>true</optimize>
					<showWarnings>true</showWarnings>
				</configuration>
			</plugin>

3) i hope that the official offer  a easy going tutorial.  
Greate Thanks!

**Comment by mickaelistria** [comment]
https://github.com/eclipse-tycho/tycho/blob/master/pom.xml#L68
https://github.com/eclipse-tycho/tycho/blob/master/pom.xml#L58
A good editor should provide you documentation for the mojo, on hover or completion; or you can jst use the usual maven way to get plugin information: `mvn help:describe -Dplugin=org.eclipse.tycho:tycho-compiler-plugin -Dgoal=compile -Ddetail`
There are some good examples int he repo here in https://github.com/eclipse-tycho/tycho/tree/master/demo and some good tutorial existing over the web.

**Comment by zhiyoutech** [comment]
> 

Greate Thanks!
